packages/modular-scripts/src/config/jest/index.ts

```diff
@@ -37,8 +37,13 @@ export default function createJestConfig(): Config.InitialOptions {
     ...defaults,
     resetMocks: false,
     transform: {
-      '^.+\\.jsx?$': 'babel-jest',
-      '^.+\\.tsx?$': 'ts-jest',
+      '^.+\\.(js|jsx|mjs|cjs)$': [
+        'babel-jest',
```

Comment:
Thinking about this it should be ```require.resolve``` so that we use the version that we depend on to make sure that no weird hoisting happens.

Reply:
Done